src/jsutils/instanceOf.js

```diff
@@ -11,7 +11,8 @@ declare function instanceOf(
 
 // See: https://expressjs.com/en/advanced/best-practice-performance.html#set-node_env-to-production
 // See: https://webpack.js.org/guides/production/
-export default process.env.NODE_ENV === 'production'
+export default typeof process !== 'undefined' &&
+process.env.NODE_ENV === 'production'
```

Comment:
@lukejacksonn Can you please extract it into separate PR?
Also, please test that Webpack is able to recognize this pattern and switch to the production code.

Is it the last change we need to support ESM?

Reply:
Yeh I can do that.. this is the last source modification required for ESM support (assuming that the mjs build now adds extensions to imports paths).

Reply:
> Yeh I can do that..

Great 👍 Thanks a lot!

> assuming that the mjs build now adds extensions to imports paths

You can see preview build here:
https://github.com/graphql/graphql-js/blob/npm/graphql.mjs
If you have time to try it here is the instruction:
https://github.com/graphql/graphql-js#want-to-ride-the-bleeding-edge

Reply:
Hi @lukejacksonn,
Did you have a time to test this change with webpack?
I planning to release new RC so it would be great to also include this change.

Reply:
Hey @IvanGoncharov sorry this task completely slipped my mind. I have made a PR https://github.com/graphql/graphql-js/pull/2409 but am not so sure about what webpack specific config needs to be checked